### ISSUE TITLE
Endless redirect after adding locale for localization

### ISSUE THREAD

**Issue by vhontar** [statamic/v2-hub#1529]
### Expected behaviour
I've tried to add localization to statamic using your documentation. I will expect that when i add new locale it will give me an opportunity to change language by url and content will change too.

### Actual behaviour
Endless redirect with 200 status.

### Steps to reproduce
1.  Add folders with languages
2.  Add folders to collection with articles for different languages
3.  Add Nginx configuration
4. Add new locale to system.yaml
--- This break down everything ---
Endless redirect, and i removed this locale from system.yaml didn't help at all.
### Server Details

**Operating System**:
Ubuntu 17.04
**Web Server:**
Nginx
**PHP Version:**
7.1.6
**Statamic Version:**
2.6.1

**Updated from an older Statamic or fresh install:**
Updated from older statamic, but I've updated statamic because of this problem, i thought that maybe updating statamic would fix it.
**List of installed addons:**
Fetch
FileAttributes

### Logs
If any logs (browser, server, or Statamic) are appropriate...
Last error before update the page and endless redirect is:
<details>
```
[2017-07-27 11:39:07] production.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function in() on null in /var/www/public_html/statamic/core/Stache/Updater.php:231
Stack trace:
#0 /var/www/public_html/statamic/vendor/laravel/framework/src/Illuminate/Support/Collection.php(135): Statamic\Stache\Updater->Statamic\Stache\{closure}('title: Una orac...', 'collections/art...')
#1 /var/www/public_html/statamic/core/Stache/Updater.php(240): Illuminate\Support\Collection->each(Object(Closure))
#2 /var/www/public_html/statamic/core/Stache/Updater.php(181): Statamic\Stache\Updater->updateLocale('es', Object(Statamic\Stache\Repository), Object(Illuminate\Support\Collection), Object(Illuminate\Support\Collection))
#3 /var/www/public_html/statamic/core/Stache/Updater.php(161): Statamic\Stache\Updater->updateLocaleAggregate('es', Object(Statamic\Stache\Repository), Object(Illuminate\Support\Collection), Object(Illuminate\Support\Collection))
#4 /var/www/public_html/statamic/core/Stache/Updater.php(113): Statamic\Stache\Updater->updateLocalized()
#5 /var/www/public_html/statamic/core/Stache/UpdateManager.php(55): Statamic\Stache\Updater->update()
#6 /var/www/public_html/statamic/vendor/laravel/framework/src/Illuminate/Support/Collection.php(135): Statamic\Stache\UpdateManager->Statamic\Stache\{closure}(Object(Statamic\Stache\Drivers\EntriesDriver), 'entries')
#7 /var/www/public_html/statamic/core/Stache/UpdateManager.php(63): Illuminate\Support\Collection->each(Object(Closure))
#8 /var/www/public_html/statamic/core/Stache/Manager.php(76): Statamic\Stache\UpdateManager->update()
#9 /var/www/public_html/statamic/core/Providers/StacheServiceProvider.php(158): Statamic\Stache\Manager->update()
#10 /var/www/public_html/statamic/core/Providers/StacheServiceProvider.php(140): Statamic\Providers\StacheServiceProvider->updateStache()
#11 [internal function]: Statamic\Providers\StacheServiceProvider->boot(Object(Illuminate\Http\Request))
#12 /var/www/public_html/statamic/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#13 /var/www/public_html/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(734): Illuminate\Container\Container->call(Array)
#14 /var/www/public_html/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(717): Illuminate\Foundation\Application->bootProvider(Object(Statamic\Providers\StacheServiceProvider))
#15 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(Statamic\Providers\StacheServiceProvider), 27)
#16 /var/www/public_html/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(718): array_walk(Array, Object(Closure))
#17 /var/www/public_html/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/BootProviders.php(17): Illuminate\Foundation\Application->boot()
#18 /var/www/public_html/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(203): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Statamic\Application))
#19 /var/www/public_html/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(222): Illuminate\Foundation\Application->bootstrapWith(Array)
#20 /var/www/public_html/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(117): Illuminate\Foundation\Http\Kernel->bootstrap()
#21 /var/www/public_html/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#22 /var/www/public_html/index.php(142): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#23 {main}  
```
</details>

**Comment by vhontar** [comment]
This method occurs this strange behaviour:
$this->outputRefreshResponse();
File: StacheServiceProvider.php

$html = sprintf('<meta http-equiv="refresh" content="1; URL=\'%s\'" />', $url);
this one, what is it? why is it here? it resolve me when i comment the method above.

**Comment by vhontar** [comment]
Interesting that i created my collections like in your documentation and articles for default English locale were in the root of folder article:
In both Collections and Taxonomies, the localized entries/terms are all stored in subfolders with identical filenames to the default locale. eg. categories/news.md and categories/fr/news.md.

I created en folder and put all content from root there and endless redirect was fixed.
